packages/components/src/Combobox/components/ComboboxOption/ComboboxOption.tsx

```diff
@@ -29,7 +31,12 @@ export function ComboboxOption(props: ComboboxOptionProps) {
       }
       className={classnames(styles.option)}
     >
-      {props.label}
+      <Flex template={["shrink", "grow"]}>
+        {props.prefix}
+
+        {props.label}
+      </Flex>
```

Comment:
this is a bit problematic on long labels when you don't have a prefix 

![image](https://github.com/GetJobber/atlantis/assets/15986172/3833c57b-c676-4a18-8530-366e7f4d4b0a)

for reference, it does this when you have a prefix

![image](https://github.com/GetJobber/atlantis/assets/15986172/f0446ccc-d3ea-44e9-b6cf-f0151ca630aa)

Reply:
Found a way to ensure the rules are applied, not sure how pretty it is though!